docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command:
+      sh -c "python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
```

Comment:
The `depends_on` directive only ensures that the `db` service is started before the `cinema` service, but it does not wait for the database to be ready to accept connections. Consider using a script or tool to wait for the database to be ready before running migrations and starting the server.